### DISCUSSION TITLE
How to make numbering skip first page?

### DISCUSSION THREAD

**Discussion by dsantiago** [typst/typst#1169] [General]
I am using this to format the page.

```rust
set page(paper: "a4", margin: (bottom: 2cm), numbering: "1")
```

but I want to skip the first page as it's my homework presentation... 
How can I do it?

**Comment by maciemesser** [comment]
How about this:
```
#set page(numbering: none)

Content of your first page

#pagebreak()
#set page(numbering: "1")

```

And if you do not only want to start numbering on the second page, but also start there with a 1 instead of a 2:

 ```
#set page(numbering: none)

Content of your first page

#pagebreak()
#set page(numbering: "1")
#counter(page).update(1)
```

**Reply by dsantiago** [reply]
Thanks for the help, it worked!

**Reply by explosion-mental** [reply]
The second part doesn't seem to work when using `number-align` ;(

Edit: specifically and only with `top` variants
